project/settings.py

```diff
@@ -320,10 +320,11 @@ def abspath(*args):
 
 ELASTICSEARCH_HOST = 'http://localhost:9200'
 UNICORE_DISTRIBUTE_HOST = 'http://localhost:6543'
+SERVICE_HOST_IP = '127.0.0.1'
```

Comment:
I'm going with a centralized services approach for now, so we'll be sharing Redis, ES and UCD. This IP is then passed to the container using `--add-host servicehost:{IP}`